R/constants-aglu-L2.R

```diff
@@ -0,0 +1,201 @@
+# Column names of tables that are read in to the model interface
+# These correspond to the headers of the same name in the headers/ModelInterface_headers.txt file
+# These are for generic categories; module-specific level2 data names are found in data header files of each module
+
+R <- "region"
+Y <- "year"
+input <- "minicam.energy.input"
```

Comment:
@pkyle what's this `input` that appears in the old data system? Does it ever change? Why does "minicam.energy.input" get used below as a string?